src/main/java/strategy/FruitOperationImpl.java

```diff
@@ -0,0 +1,21 @@
+package strategy;
+
+import dao.Storage;
+import model.FruitTransaction;
+
+public class FruitOperationImpl implements FruitOperation {
```

Comment:
bad class name - generic. We need to read `execute` method to understand its role